docs/user_guide/authenticate.md

```diff
@@ -6,5 +6,6 @@ Introduces the `earthaccess.login` method for managing Earthdata Login and cloud
 
     We are reorganizing and updating the documentation, so not all pages are complete.  If you are looking for information about authenticating using earthaccess see the
     How-Tos and Tutorials in links below.
-    [How-To Authenticate with earthaccess](/howto/authenticate.md)
-    [Getting Started Tutorial](/tutorials/getting-started.ipynb)
+
+    * [Quick start](../../quick-start/)
```

Comment:
The "getting started" tutorial doesn't appear to exist anymore, so I linked to the 'quick-start'.